tests/testthat/test-ipc.R

```diff
@@ -0,0 +1,9 @@
+test_that("Test reading data from ipc", {
+  tmpf = tempfile()
```

Comment:
```suggestion
  tmpf = tempfile()
  on.exit(unlink(tmpf))
```

Reply:
Fixed.

I also notice some other unit tests using `tempfile()` missing `on.exit(unlink(...))`. Should I update them as well?

Reply:
> I also notice some other unit tests using `tempfile()` missing `on.exit(unlink(...))`. Should I update them as well?

Thanks for finding it.
If you are willing to take on that fix, another PR would be fine. (We can merge this PR without waiting for that fix)

Reply:
I'll make another PR for the fix